src/imitation/util/sacred.py

```diff
@@ -67,24 +71,33 @@ def filter_subdirs(
     return list(filtered_dirs)
 
 
-def build_sacred_symlink(log_dir: str, run: sacred.run.Run) -> None:
+def build_sacred_symlink(log_dir: types.AnyPath, run: sacred_run.Run) -> None:
     """Constructs a symlink "{log_dir}/sacred" => "${SACRED_PATH}"."""
+    log_dir = pathlib.Path(log_dir)
+
     sacred_dir = get_sacred_dir_from_run(run)
     if sacred_dir is None:
         warnings.warn(RuntimeWarning("Couldn't find sacred directory."))
         return
-    symlink_path = os.path.join(log_dir, "sacred")
-    target_path = os.path.relpath(sacred_dir, start=log_dir)
+    symlink_path = pathlib.Path(log_dir, "sacred")
+    target_path = pathlib.Path(os.path.relpath(sacred_dir, start=log_dir))
+
+    # Path.symlink_to errors if the symlink already exists. In our case, we actually
+    # want to override the symlink to point to the most recent Sacred dir. The
+    # examples/quickstart.sh script fails without this check when run a second time.
+    if symlink_path.is_symlink():
+        symlink_path.unlink()
+
     # Use relative paths so we can mount the output directory at different paths
     # (e.g. when copying across machines).
-    os.symlink(target_path, symlink_path, target_is_directory=True)
+    symlink_path.symlink_to(target_path, target_is_directory=True)
```

Comment:
Potential race condition here if something else creates a symlink in the background, but I don't think that matters too much and can't see a way to avoid it using pathlib.